index-metagenomics.md

```diff
@@ -1,53 +1,129 @@
 
-* <a target="_parent" href="https://metagenomics.usegalaxy.eu/tours/metagenomics-general-tutorial-shotgun">Analyses of metagenomics data - Shotgun metagenomics data analyses</a>: In this tour, we will use analyze shotgun datasets from soil samples.
+    This tutorial introduces the amplicon and shotgun data analyses with the general principles behind and the differences.
 
+- [16S Microbial Analysis with Mothur](https://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.html)
 
-## Galaxy for Metagenomics -- many possibilities 
+    In this tutorial the Standard Operating Procedure (SOP) for MiSeq data, developed by the creators of the Mothur software package, is perfomed within Galaxy.
 
-* 16s RNA
-* Genome assembly
-* ...
+<a name="workflows"></a>
+## Workflows
 
+To orchestrate tools and help users with their analyses, several [workflows](http://asaim.readthedocs.io/en/latest/workflows.html) are available. They formally orchestrate tools in a defined order and with defined parameters, but they are customizable (tools, order, parameters).
 
+The workflows are available in the [Shared Workflows](https://metagenomics.usegalaxy.eu/workflows/list_published), with the label "***asaim***".
 
-## Workflows
+### Analysis of raw metagenomic or metatranscriptomic shotgun data
+
+The workflow quickly produces, from raw metagenomic or metatranscriptomic shotgun data, accurate and precise taxonomic assignations, wide extended functional results and taxonomically related metabolism information
+
+![](http://asaim.readthedocs.io/en/latest/_images/main_workflow.png)
+
+This workflow consists of
+
+1. Processing with quality control/trimming (**FastQC** and **Trim Galore!**) and dereplication (**VSearch**)
+2. Taxonomic analyses with assignation (**MetaPhlAn2**) and visualization (**KRONA**, **GraPhlAn**)
+3. Functional analyses with metabolic assignation and pathway reconstruction (**HUMAnN2**)
+4. Functional and taxonomic combination with developed tools combining HUMAnN2 and MetaPhlAn2 outputs
+
+It is available with 4 versions, given the input
+
+1. Simple files: [Single-end](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim-shotgun-workflow) or [paired-end](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim---shotgun-workflow-for-paired-end-data)
+2. Collection input files: [Single-end](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim-shotgun-workflow-se-collection) or [paired-end](https://metagenomics.usegalaxy.eu/u/berenice/w/asaim---shotgun-workflow-for-paired-end-data-collection)
+
+### Assembly of metagenomic data
 
+To reconstruct genomes or to get longer sequences for further analysis, microbiota data needs to be assembled, using the recently developed metagenomics assemblers.
 
+To help in this task, two workflows have been developed in ASaiM, each one using one of each of the well-performing assemblers:
```

Comment:
`To help in this task, two workflows have been developed using two different well-performing assemblers:`